src/components/icon/GraphItemIcon/GraphItemIcon.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-
+import { AppIcon } from '@folio/stripes/core';
 import {
```

Comment:
```suggestion
import { AppIcon } from '@folio/stripes/core';

import {
```

The style practice we are following (and need to document) is to separate the types of imports by new lines.

The local files should have a new line.